guides/source/configuring.md

```diff
@@ -648,6 +648,16 @@ deploying to a memory constrained environment you may want to set this to `false
 | 7.2                   | `true`               |
 | 8.1                   | `!Rails.env.local?`  |
 
+#### `config.yjit_options`
+
+Allows you to set options for YJIT. The default value is an empty hash.
+When `config.yjit` is `false`, this setting will be ignored.
+
+```ruby
+config.yjit = true
+config.yjit_options = { stats: true }
```

Comment:
I think we could simply allow:

```suggestion
config.yjit = { stats: true }
```

Reply:
Thank you! As you pointed out, I simplified it by allowing a hash.